packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/NotificationSerializer.java

```diff
@@ -66,12 +61,11 @@ public static Bundle toBundle(NotificationRequest request) {
     if (existingContentData == null) {
       if(requestTrigger instanceof FirebaseNotificationTrigger trigger) {
         RemoteMessage message = trigger.getRemoteMessage();
-        RemoteMessage.Notification notification = message.getNotification();
         Map<String, String> data = message.getData();
         String dataBody = data.get("body");
-        String notificationBody = notification != null ? notification.getBody() : null;
-        if (isValidJSONString(dataBody) && notificationBody != null && notificationBody.equals(data.get("message"))) {
```

Comment:
this condition was too strict - it required remote notification to have a body string (that's the string rendered below the title)